controls/badgeview/badge-overview.md

```diff
@@ -10,7 +10,7 @@ slug: badge-overview
 
 The Badge is the indicator (or marker) which is displayed to show notifications, statuses, and so on, while the BadgeView is the control in which you can add content and position the Badge based on this content.
 
->important To provide the best experience out of the BadgeView control and the Badge indicator, it is recommended that you use the [BadgeView]({%slug badgeview-overview%}) and customize the Badge whenever needed.
+>important To provide the best experience out of the BadgeView control and the Badge indicator, it's recommended that you use the [BadgeView]({%slug badgeview-overview%}) and customize the Badge whenever needed.
```

Comment:
I find this note very confusing. Can you rephrase and clarify it?
It sounds like "For the best experience with the Badge and BadgeView, customize the Badge and use the BageView," which is nonsense.

Reply:
description was confusing, i have changed it